fw/sock.c

```diff
@@ -1414,14 +1456,19 @@ ss_tx_action(void)
 
 		bh_lock_sock(sk);
 		if (sock_flag(sk, SOCK_DEAD)) {
```

Comment:
What about this comment https://github.com/tempesta-tech/tempesta/pull/2007/files#r1406949163 ? At least a code comment must be added

Reply:
I add comment about